administrator-manual/en/fail2ban.rst

```diff
@@ -0,0 +1,160 @@
+========
+Fail2ban
+========
+
+Fail2ban scans log files (e.g. /var/log/apache/error_log) and bans IPs that show the malicious signs – too many password failures, seeking for exploits, etc. Generally Fail2Ban is then used to update firewall rules to reject the IP addresses for a specified amount of time, although any arbitrary other action (e.g. sending an email) could also be configured. Out of the box Fail2Ban comes with filters for various services (apache, courier, ssh, etc).
+
+Fail2Ban is able to reduce the rate of incorrect authentications attempts however it cannot eliminate the risk that weak authentication presents. Configure services to use only two factors or public/private authentication mechanisms if you really want to protect services.
```

Comment:
Add a comma after the word "attempts".

Also, I'd change this:
"Configure services to use only two factors or public/private authentication mechanisms if you really want to protect services." 
To something like:
"To improve the security, open the access to service only for secure networks using the firewall."